src/o365/onedrive/commands/report/onedrive-report-activityfilecounts.spec.ts

```diff
@@ -0,0 +1,95 @@
+import commands from '../../commands';
```

Comment:
Following the naming convention, the file should be named `report-activityfilecounts.spec.ts`